package.json

```diff
@@ -34,10 +34,10 @@
     "sinon": "^7.3.2",
     "ts-node": "^8.3.0",
     "typedoc": "^0.15.0",
-    "typescript": "^3.5.3"
+    "typescript": "^3.8.0"
```

Comment:
Should this be called out in CHANGELOG?

Reply:
It's a dev dependency so it doesn't matter to users